SWIG_CGAL/common.i

```diff
@@ -279,7 +279,7 @@
 //exception for iterators (for next functions)
 #ifdef SWIGPYTHON
 %include exception.i
-%exception next
+%exception
```

Comment:
In that case, I think the exception should only apply to __next__ and/or next to avoid pollution in the code generated by SWIG